lib/macho/exceptions.rb

```diff
@@ -127,8 +127,8 @@ class HeaderPadError < MachOError
     # @param filename [String] the filename
     def initialize(filename)
       super "Updated load commands do not fit in the header of " +
-      "#{filename}. #{filename} needs to be relinked, possibly with " +
-      "-headerpad or -headerpad_max_install_names"
+        "#{filename}. #{filename} needs to be relinked, possibly with " +
+        "-headerpad or -headerpad_max_install_names"
```

Comment:
Sounds good!

I ignored that suggestion originally because I generally shy away from metasyntax like line continuations, but best practice is best practice :smile: